recurser/string-is
# Add a Markdown formatter

Replace this with a detailed explanation of the change

## How to test the PR

Paste some markdown and check that the format is auto-detected, and the formatting works correctly.

## Checklist

None
